src/middlewared/middlewared/alembic/versions/25.04/2024-10-29_12-43_vm_zvol_serial.py

```diff
@@ -0,0 +1,31 @@
+"""
+Add serial to VM zvol disk devices
+
+Revision ID: a4ce4939b908
+Revises: dd6e581235b2
+Create Date: 2024-10-29 12:43:45.960915+00:00
+
+"""
+import json
+
+from alembic import op
+
+from middlewared.utils.crypto import generate_string
```

Comment:
Please just copypaste the function here. We should not import anything in migrations code, they should be immutable.